public/docs/_examples/setup/ts/app/main.ts

```diff
@@ -0,0 +1,7 @@
+// #docregion
+import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
+import { AppModule }              from './app.module';
+platformBrowserDynamic().bootstrapModule(AppModule);
+// #enddocregion
+
+/* This file is excluded in plunkers */
```

Comment:
Nit: if you put this comment on the first line then you can delete lines 5 and 6.